website/docs/core-concepts/stacks/templates/functions/atmos.Vals.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: atmos.Vals
+sidebar_position: 3
+sidebar_label: atmos.Vals
+description: Load config values and secrets from external sources
+---
+import File from '@site/src/components/File'
+import Intro from '@site/src/components/Intro'
+
+<Intro>
+The `atmos.Vals` template function wraps [vals](https://github.com/helmfile/vals) to support loading config values and secrets from external sources.
+</Intro>
+
+## Usage
+
+```yaml
+  {{ atmos.Vals "<reference>" }}
+```
+
```

Comment:
We should also add a note about how the authentication is handled by vals, and that environment variables need to be set corresponding to the secrets backend used.